Homework9_2.py

```diff
@@ -0,0 +1,30 @@
+class PizzaError(Exception):
+    def __init__(self,pizza = 'unknown', message = ''):
+        Exception.__init__(self,pizza,message)
+        self.pizza = pizza
+
+class TooMuchCheese(PizzaError):
+    def __init__(self, pizza = 'unknown', cheese = '>100', message = ''):
+        PizzaError.__init__(self,pizza,message)
+        self.cheese = cheese
+
+class Pizza:
+    def __init__(self):
+        pass
+    
+    def make_pizza(self,pizza,cheese):
+        if pizza not in ['margherita','capricciosa','calzone']:
+            raise PizzaError
+        if cheese > 100:
+            raise TooMuchCheese
+        print(pizza,"pizza is ready!")
```

Comment:
No, no, no)) 
This trick with skipping the constructor is a bad practice, we use the constructor not to formally use it, but to create a list of pizzas at least in this case.

Try to filling the next statements:

```suggestion
class Pizza():
    def _....__(self):
        self.__pzz_list = ["................................"]
    def make_pizza(self, pizza, cheese):
        self.pizza = pizza
        self.cheese = ....................
        if self.pizza not in self.__pz_list:
            raise PizzaError (self.pizza, "pizza not in menu")
        if .......cheese > 100:
            raise TooMuchCheeseError (self.pizza, self.ch.........., "to much cheese")
        print("Pizza ready!")
```